jdi-light-mobile-examples/src/main/java/selenium/seleniumPO/PerformancePage.java

```diff
@@ -0,0 +1,45 @@
+package selenium.seleniumPO;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
+
+public class PerformancePage {
+    //Table
+    @FindBy(id = "users-table") public WebElement table;
+
+    public WebElement getUser(String name, String email) {
+        List<WebElement> rows = table.findElements(By.tagName("tr"));
+        for (WebElement row : rows) {
+            List<WebElement> columns = row.findElements(By.tagName("td"));
+            if (columns.size() == 4 && columns.get(0).getText().contains(name) &&
+                    columns.get(2).getText().contains(email))
+                return row;
+        }
+        return null;
+    }
+
+    // Droplist
+    @FindBy(css = "select[ui=dropdown]")
+    private WebElement dropdownValue;
+    @FindBy(css = "#user-names option")
+    private List<WebElement> dropdown;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)